studio/components/interfaces/SQLEditor/SQLEditor.constants.ts

```diff
@@ -1256,4 +1256,4 @@ export const sqlAiDisclaimerComment = stripIndent`
 
 export const untitledSnippetTitle = 'Untitled query'
 
-export const destructiveSqlRegex = [/drop/i, /delete/i, /truncate/i]
+export const destructiveSqlRegex = [/(drop|delete|truncate) /i]
```

Comment:
Specifically - adding the space ` ` at the end should reduce a lot of false positives.

Reply:
Updated to use `\s` instead to account for line breaks, tabs, etc.